tutorials/llm/llama-3/pruning-distillation/05_display_results.ipynb

```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6c91263b-b312-4ab2-b13f-0ee4b6e8bd0f",
+   "metadata": {},
+   "source": [
+    "### Step 5: Display the validation loss\n",
+    "\n",
+    "Now that the results are in, let's visualize the validation loss of the two distilled models using the `tensorboard` library. \n",
+    "> `NOTE:` This notebook demonstrates the use of the teacher finetuning, pruning and the distillation script. These scripts should ideally be run on a multi-node cluster with a larger `GLOBAL_BATCH_SIZE` and `STEPS` to see improvement in the validation loss."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5822d62-8131-4046-8c22-0bf0fce81df7",
+   "metadata": {},
+   "source": [
+    "#### Validation Loss using depth-pruned model as student in distillation script\n",
+    "Here is an image of the validation loss over 30 steps of running the training step in the distillation script when we distill the knowledge from the finetuned teacher model to the depth-pruned student."
```

Comment:
fix punctuation, revise sentence

"Here is an image of the validation loss over 30 steps of running the training step in the distillation script, where we distill the knowledge from the fine-tuned teacher model to the depth-pruned student."